.github/workflows/test.yml

```diff
@@ -81,3 +81,21 @@ jobs:
 
       - name: Run javascript tests
         run: npm test
+
+  deploy:
+    uses: "./.github/workflows/deploy-generic.yml"
+    with:
+      env: "test"
+      ecr_repository: ${{ vars.TEST_ECR_REPOSITORY }}
+      ecr_region: ${{ vars.TEST_ECR_REGION }}
+      debug: ${{ vars.DEBUG }}
+      django_allowed_hosts: ${{ vars.DJANGO_ALLOWED_HOSTS }}
+      catalogue_url: ${{ vars.CATALOGUE_URL }}
```

Comment:
These don't need to be passed as variables, as they'll be available to the workflow's environment by default.

The only reason to pass them is if you want _different_ variables than would exist in the workflow's environment. E.g. if you wanted _repository_ level variables to be used when an _environment_-specific variable exists with the same name. This would probably need explicit callout as well on top of the definition here.

See [github docs for more details](https://docs.github.com/en/actions/learn-github-actions/variables#defining-configuration-variables-for-multiple-workflows):
>When you define configuration variables, they are automatically available in the vars context. For more information, see "Using the vars context to access configuration variable values".